crates/stages/src/stages/sender_recovery.rs

```diff
@@ -178,6 +154,24 @@ impl<DB: Database> Stage<DB> for SenderRecoveryStage {
     }
 }
 
+fn recover_sender(
```

Comment:
no idea if this is beneficial tbh
```suggestion
#[inline]
fn recover_sender(
```

Reply:
not sure that inlining is beneficial here